lib/appcues_flutter.dart

```diff
@@ -101,6 +114,10 @@ class Appcues {
         '_dartVersion': Platform.version
       }
     });
+
+    // consider whether this should be opt-in or configurable
+    RendererBinding.instance.pipelineOwner
+        .ensureSemantics(listener: _semanticsChanged);
```

Comment:
this is the key spot where we hook in for Semantics tree updates - it is unclear what type of performance impact this might have on a large application, need to understand more about that and what sort of options we could provide.

Reply:
If we're worried about performance, doing the work in the background might be a good idea?

https://docs.flutter.dev/cookbook/networking/background-parsing
https://dart.dev/language/concurrency

Reply:
hitting snags here on initial attempts unfortunately. The Dart `compute` function does not allow passing certain Flutter types (like `SemanticsNode`) 
```
Invalid argument(s): Illegal argument in isolate message: (object extends NativeWrapper
```

nor can you access other Flutter specific things like the `RendererBinding` from within the compute function
```
Unhandled Exception: Binding has not yet been initialized.
The "instance" getter on the RendererBinding binding mixin is only available once that binding has been initialized.
```

Will have to look into this one further to see if there is any way we can push these things to the background.

Reply:
So far I've been unable to find a workable solution to push this work to a background thread, unfortunately.

To start, made this update https://github.com/appcues/appcues-flutter-plugin/pull/43/commits/9436229c16b988b25e7b064fecbe0200532e69ee which will allow the host app to have full control over when they want to opt in or out of Semantics tree observation, to hopefully alleviate any issues. It will start disabled, no change from the current plugin, but you can enable with the one-liner anywhere in the app to begin supporting element targeting.
```
Appcues.enableElementTargeting();
```